demoapp/composeApp/src/commonMain/kotlin/at/asitplus/cryptotest/App.kt

```diff
@@ -94,7 +94,7 @@ val SIGNER_CONFIG: (SignerConfiguration.()->Unit) = {
 
 val context = newSingleThreadContext("crypto").also { Napier.base(DebugAntilog()) }
 
-private class getter<T>(private val fn: ()->T) {
+private class getter<T>(private val fn: () -> T) {
```

Comment:
i prefer the compact form since it visually binds the typename `()->T` together, but am willing to be convinced

Reply:
formatting convention